cli/src/commands/staking/validate.ts

```diff
@@ -0,0 +1,37 @@
+import StakingCommandBase from '../../base/StakingCommandBase'
+import { flags } from '@oclif/command'
+
+
+export default class StakingValidateCommand extends StakingCommandBase {
+  static description = 'Start validating. Takes the controller key.'
+  static args = [
+    {
+      name: 'commission',
+      required: false,
+      description: 'Set a commission (0-100), which is deducted from all rewards before the remainder is split with nominator',
+    },
+  ]
+  static flags = {
+    controller: flags.string({
+      required: false,
+      description: `The controller key you want to validate with.`,
+    }),
+  }
+  
+  async run(): Promise<void> {
+    let { commission } = this.parse(StakingValidateCommand).args
+    let { controller } = this.parse(StakingValidateCommand).flags
+    await this.electionStatus()
+    const validatorPrefs = await this.getValidatorPrefs(commission)
+
+    if (controller === undefined) {
+      controller = await this.getController()
+    } else {
+      await this.isController(controller)
+    }
+    await this.electionStatus()
```

Comment:
Probably not, but for (future) commands that may take a little more time for the user to complete, it would probably be good to do one before they start. 

I'm not sure how useful the error message will be if it changed while the user is doing their thing, so I added an extra run at the end to "ensure" they get the message clearly.

Ideally, I guess I should calculate approximately when the `staking.*` commands will be "enabled" again, and rather just warn at initiation.